### PR TITLE
Add a model config generator

### PR DESCRIPTION
**Pull request by nithinsubbiah**
Model config generator takes a PyTorch model as input and generates a JSON file with model layers and other propperties that define sharding on a particular hardware.

### CODE REVIEW

File: apps/language_models/src/model_wrappers/vicuna_model.py [hunk 1]
```diff
@@ -237,3 +235,21 @@ def forward(
             return_vals.append(item[0])
             return_vals.append(item[1])
         return tuple(return_vals)
+
+class CombinedModel(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+        self.first_vicuna = FirstVicuna("TheBloke/vicuna-7B-1.1-HF")
+        self.second_vicuna = SecondVicuna("TheBloke/vicuna-7B-1.1-HF")
```
Comment (powderluv) [inline apps/language_models/src/model_wrappers/vicuna_model.py:244]: this can't be hardcoded right ?

Comment (nithinsubbiah) [reply apps/language_models/src/model_wrappers/vicuna_model.py]: fixed

**Comment by powderluv** [comment]
lint please

**Review by monorimet — APPROVED** [review]
LGTM.

**Pull request merged**